clusterloader2/pkg/framework/client/objects.go

```diff
@@ -264,6 +271,10 @@ func CreateObject(dynamicClient dynamic.Interface, namespace string, name string
 	gvk := obj.GroupVersionKind()
 	gvr, _ := meta.UnsafeGuessKindToResource(gvk)
 	obj.SetName(name)
+
+	ti0 := time.Now()
+	obj.SetAnnotations(map[string]string{CreateTimestampAnnotation: ti0.Format(TimestampLayout)})
```

Comment:
Correct me if I'm wrong but this seems to set Annotation for all objects and actually doesn't require "  annotations: {{.Annotations}}" as specified in the reviewers notes.

Reply:
you're correct. That was a wrong statement in the reviewers notes - I just update it.